docs/experiment/ppl/cmd/rare.rst

```diff
@@ -0,0 +1,78 @@
+=============
+rare
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| Using ``rare`` command to find the least common tuple of values of all fields in the field list.
+
+
+Syntax
+============
+rare <field-list> [by-clause]
+
+
+* field-list: mandatory. comma-delimited list of field names.
+* by-clause: optional. one or more fields to group the results by.
+
+
+Example 1: Find the least common values in a field
+===========================================
+
+The example finds least common gender of all the accounts.
+
+PPL query::
+
+    od> source=accounts | rare gender;
+    fetched rows / total rows = 2/2
+    +------------+
+    | gender     |
+    |------------|
+    | F          |
+    |------------|
+    | M          |
+    +------------+
+
+
+Example 2: Find the least common values organized by gender
+====================================================
+
+The example finds least common age of all the accounts group by gender.
+
+PPL query::
+
+    od> source=accounts | rare age by gender;
```

Comment:
how many value for each field will be returened? 10?

Reply:
10 values per group for each field

Reply:
could you add this to the doc? I think the max value = 10, right?

Reply:
Yes, the max is 10. Added in the Description section.